src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -0,0 +1,157 @@
+
+Although many browsers report mixed content warnings to the user,
+by the time this happens, it is too late:
+the insecure requests have already been performed and the security of the page is compromised.
+
+This is why mixed content is increasingly being blocked by browsers.
+If you have mixed content on your site,
+then fixing it will ensure the content continues to load as browsers become more strict.
+
+## The two types of mixed content
+
+The two types of mixed content are: active and passive.
+
+**Passive mixed content** refers to content that doesn't interact with the rest of the page,
+and thus a man-in-the-middle attack is restricted to what they can do if they intercept or change that content.
+Passive mixed content includes images, video, and audio content,
+along with other resources that cannot interact with the rest of the page.
+
+**Active mixed content** interacts with the page as a whole and allows an attacker to do almost anything with the page.
+Active mixed content includes scripts, stylesheets, iframes,
+and other code that the browser can download and execute.
+
+### Passive mixed content
+
+Passive mixed content is seen as less problematic yet still poses a security threat to your site and your users.
+For example, an attacker can intercept HTTP requests for images on your site and swap or replace these images;
+the attacker can swap the _save_ and _delete_ button images,
+causing your users to delete content without intending to;
+replace your product diagrams with lewd or pornographic content,
+defacing your site; or replace your product pictures with ads for a different site or product.
+
+Even if the attacker doesn't alter the content of your site,
+an attacker can track users via mixed content requests.
+The attacker can tell which pages a user visits and which products they view based on images or other resources that the browser loads.
+
+The following Glitch demo contains examples of passive mixed content.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/passive-mixed-content?path=index.html&previewSize=100"
+    title="passive-mixed-content on Glitch"
+    allow="encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+If passive mixed content is present most browsers will indicate in the URL bar that the page is not secure,
+even when the page itself was loaded over HTTPS.
+
+Until recently passive mixed content was loaded in all browsers,
+as to block it would have broken many websites.
+This is now beginning to change and so it is vital to update any instances of mixed content on your site.
+
+Chrome is currently rolling out automatic upgrading of passive mixed content where possible.
+Automatic upgrading means that if the asset is available over HTTPS, but has been hardcoded as HTTP,
+the browser will load the HTTPS version. If no secure version can be found the asset will not load.
```

Comment:
```suggestion
As of September 2020 Chrome is currently rolling out automatic upgrading of
passive mixed content where possible. Automatic upgrading means that if the
asset is available over HTTPS, but has been hardcoded as HTTP, the browser
will load the HTTPS version. If no secure version can be found, Chrome will
not load the asset.
```
Do we have a Chrome link announcing the roll-out? Do we have info on other browsers?

(`As of September 2020` and rephrasing the last sentence to avoid passive voice are the only changes here. I just wrapped the sentences to be well-below 100 characters width while I was in the neighborhood.)